product_variant_weight/tests/test_product_variant_weight.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.tests.common import TransactionCase
+
+
+class TestProductVariantWeight(TransactionCase):
+
+    def setUp(self):
+        super(TestProductVariantWeight, self).setUp()
+        self.template = self.env['product.template']
+        self.product_template = self.template.create({
+            'name': 'Product',
+            'weight': 50,
+            'weight_net': 50,
+        })
+        self.product = self.env.ref('product.product_product_4')
```

Comment:
Don't use demo data, but create yours. This way, you don't depend on this data, that may vary over versions.